source/Ship.cpp

```diff
@@ -735,8 +736,10 @@ string Ship::FlightCheck() const
 		return "limited turn?";
 	if(energy - .8 * solar < .2 * (turnEnergy + thrustEnergy))
 		return "solar power?";
-	if(!hyperDrive && !jumpDrive && !canBeCarried)
+	if(!canBeCarried && !hyperDrive && !jumpDrive)
```

Comment:
Doesn't seem necessary to re-order these conditions.

Reply:
Could "justify" it by joining the checks:
```
if(!canBeCarried)
{
    if(!hyperdrive && !jumpdrive)
        return "no hyperdrive?";
    if(fuel < JumpFuel())
        return "no fuel?"
}
```